src/components/About.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+
+class About extends Component {
+  render() {
+    return (
+        <div className="about" id="main">
```

Comment:
Your div here has a different alignment than used in ComingSoon (i.e. in other components all the child elements are indented)